_pkgdown.yml

```diff
@@ -28,7 +28,7 @@ reference:
       - orderly_list_src
   - title: From within a running report
     desc: >-
-      These are the functions that get called from your `orderly.R` file
+      These are the functions that get called from your `<reportname>.R` file
```

Comment:
I think we need a nicer name for this.  Perhaps calling it your "orderly file" or "entrypoint file" everywhere would make it nicer to read?

Reply:
yh i think ill do entrypoint file? will be consistent with our variable naming!

Reply:
although entrypoint file is a bit difficult to understand out of context, maybe for docs we go for orderly file? might be clearer for the users